### PR TITLE
Created Update CHANGELOG action

### PR DESCRIPTION
**Pull request by octoper**
I created Update CHANGELOG action to keep website documentation changelog page based on the Framework CHANGELOG.md as mentioned by @nunomaduro (Missing changelog on the website #34)
to make it work you must create a secret `CHANGELOG_KEY` to the pestphp/pest repository and fill it with a personal token with these permissions checked
image

### CODE REVIEW

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add file to filesystem
+      run: |
+        git clone https://github.com/pestphp/website pestphp-website
+        cp CHANGELOG.md pestphp-website/source/docs
+      shell: bash
```
Comment (owenvoke) [inline .github/workflows/changelog.yml:18]: Just wondering if it's worth using the `actions/checkout` action to clone the website repository for consistency. 🙂 And for copying the changelog, it doesn't require the `shell` key as this is the default on `ubuntu-latest`. 👍
```suggestion
    - uses: actions/checkout@v2
    - name: Checkout website repository
      uses: actions/checkout@v2
      with:
        token: ${{ secrets.CHANGELOG_KEY }}
        repository: pestphp/website
        path: website
    - name: Copy CHANGELOG to website repository
      run: cp CHANGELOG.md website/source/docs
```

Comment (octoper) [reply .github/workflows/changelog.yml:18]: Yeah have you tested it?

Comment (owenvoke) [reply .github/workflows/changelog.yml:18]: Yep, I've just done a test (also updated my suggestion comment):
- [Test repo with the action workflow](https://github.com/owenvoke/changelog-test-action)
- [Test repo to emulate the website repo](https://github.com/owenvoke/changelog-test)

Here's the action run: https://github.com/owenvoke/changelog-test-action/runs/735251194

And the PR: https://github.com/owenvoke/changelog-test/pull/1

**Comment by nunomaduro** [comment]
@octoper Can you explain me this on discord?

**Comment by owenvoke** [comment]
I think it might be worth applying the changes I mentioned to this 👍 It provides a bit more clarity and has better error handling.

**Pull request merged**